intel_extension_for_transformers/llm/runtime/graph/scripts/quantize.py

```diff
@@ -82,8 +82,8 @@ def main(args_in: Optional[List[str]] = None) -> None:
         "--compute_dtype",
         type=str,
         choices=["fp32", "fp16", "bf16", "int8"],
-        help="Data type of Gemm computation: int8/bf16/fp32 (default: int8)",
-        default="int8",
+        help="Data type of Gemm computation: int8/bf16/fp32 (default: fp32)",
+        default="fp32",
```

Comment:
Unaligned pybind/executable behavior? 

https://github.com/intel/intel-extension-for-transformers/blob/c02dd7b69f6faaafcbd394ec5c8996c63d5cdb52/intel_extension_for_transformers/llm/runtime/graph/application/common.h#L145

https://github.com/intel/intel-extension-for-transformers/blob/c02dd7b69f6faaafcbd394ec5c8996c63d5cdb52/intel_extension_for_transformers/llm/runtime/graph/application/common.cpp#L655

Reply:
That nice catch, let me continue refining it. Thanks.

Reply:
[9b6f255](https://github.com/intel/intel-extension-for-transformers/pull/945/commits/9b6f2554c0193910f4888120d12c16ef82c0bcc4)